lisa/transformers/tool_installer.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema, tools
+from lisa.executable import Tool
+from lisa.node import quick_connect
+from lisa.transformer import Transformer
+from lisa.util import LisaException, field_metadata
+
+
+@dataclass_json
+@dataclass
+class ToolInstallerTransformerSchema(schema.Transformer):
+    tool_names: List[str] = field(default_factory=List[str])
+    # the SSH connection information to the node
+    connection: Optional[schema.RemoteNode] = field(
+        default=None, metadata=field_metadata(required=True)
+    )
+
+
+class ToolInstallerTransformer(Transformer):
```

Comment:
What is this used for? Can it be a test case in the internal repo?

Reply:
Often, we verify VM behavior by manually executing the commands instead of running the automated test case. Mostly when trying to debug an issue found while executing a test or for performance related tests. This Transformer can be used in such cases to install the required tools in a repro VM, and then manually execute the commands.

I did not understand what you meant by "can it be a test case". I can ping you offline if more discussion is required.

Reply:
The similar logic can be in a test case, the test case is to install the tool. If it's not to setup something for every test case, it doesn't need to use transformer. The test case provides more rich functionality, like to run with combinators, set req and others. The env can be kept by keep_environment:always